instrumentation/rabbitmq-2.7/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/rabbitmq/ReactorRabbitMqTest.java

```diff
@@ -33,33 +40,24 @@ void testShouldNotFailDeclaringExchange() {
     Sender sender =
         RabbitFlux.createSender(new SenderOptions().connectionFactory(connectionFactory));
 
-    try {
-      sender.declareExchange(ExchangeSpecification.exchange("testExchange")).block();
-    } catch (RuntimeException e) {
-      Assertions.fail("Should not fail declaring exchange", e);
-    }
```

Comment:
these changes to the test itself are just cleanups and refactoring to use our more standard approach of assertions